### ISSUE TITLE
CellEditingStopped in fullRow Edit Mode

### ISSUE THREAD

**Issue by infornite** [avallete/ag-grid-autocomplete-editor#47]
I am editing a row in ag-grid with editType=fullrow. This works fine for the cell editors that are packaged with ag-grid but I am having a problem with the AutoComplete editor. 

When I click on a selection then the cellEditingStopped and rowEditingStopped functions are called. The result is that my row is submitted before I am ready and the other rows have been completed,

I'm quiet new to Angular and Ag-Grid so I am not sure if this is a bug or my misunderstanding of how it should work.

Thanks

**Comment by avallete** [comment]
Hi there !

I'm afraid that 'fullrow' editing, is not, and will probably not be available for ag-grid-autocomplete-editor.

Because, this cellEditor is an 'PopupEditor' (like `agPopupSelectCellEditor` is) and that, it is not recommended by ag-Grid itself to make this kind of editor work with 'fullrow' editing (see [ag-grid-docuumentation about that here](https://www.ag-grid.com/javascript-grid-cell-editing/#fullRowEdit)).

TLDR:
```
Full row editing is not compatible with popup editors.
```

However, I know that `SelectCellEditor` work with `fullRowEdit` probably due to the fact that it use the native browser `select` element which handle a lot by itself. So we may probably be able to make `autocompleteEditor` work as well with fullRowEdit.

But it will probably be very tricky and require some change in both `autocompleter` subpackage (and the way it handle lot of things with global DOM attributes) and `ag-grid-autocomplete-editor.ts` class itself.

I will probably don't work on it anytime soon, but if someone make it work with this edit mode without too much dirty hacks, I will consider merging it.

**Comment by infornite** [comment]
Thanks for the fast response. Its a great component so thanks for all your work on this.

In my case all of my grids are in fullRowEdit. Is there a way (even if its a bit hacky) that I can prevent it calling cellEditingStopped? I tried to intercept the onSelect but to no avail.

**Comment by avallete** [comment]
You will probably face some other side effects even if you are able to bypass this one (the first I can think of is if you have multiples AutocompleteCellEditor on a same row)

But on your question precisely, I would dig onto the way `destroy` function work:

```js
...
    destroy(): void {
        this.focusOut();
        if (this.stopEditing) {
            this.stopEditing();
        }
    }
```

Maybe removing the call to `stopEditing` will did the trick for you.
